.github/workflows/release.yml

```diff
@@ -3,27 +3,22 @@ name: Release
 on:
   release:
     types: [published]
+    
+env:
+  LLVM_SHA: 0b5eb7ba7bd796fb39c8bb6acd9ef6c140f28b65
+  LLVM_TARGET: x86_64-unknown-linux-musl
 
 jobs:
-  llvm:
-    uses: ./.github/workflows/llvm.yml
-
   upload-bins:
     # TODO: Build for macos someday.
     runs-on: ubuntu-22.04
-    needs: llvm
```

Comment:
we should keep this plumbing the same. it's the llvm job that should try to fetch from rustc and if it fails then it builds from src